src/utils/__tests__/actions.spec.js

```diff
@@ -585,7 +585,7 @@ describe("actions", () => {
         isEnabled: jest.fn().mockReturnValue(true),
       };
       const result = getActionsList(shouldHide);
-      expect(result.length).toBe(13);
+      expect(result.length).toBe(14);
```

Comment:
What does `feature blacklisted navigations` means? is cyclic group one of the feature blacklisted navigations?

Reply:
This is also for apps running in Direct Query mode. The difference with unsupported features is that this is controlled by the client. Mostly bigger parts that just do not make sense when the app is DQ